Updated formatting in "Persistent Volumes" & "Storage Classes" (#19575)
kubernetes/website: content/en/docs/concepts/storage/storage-classes.md

### Provisioner

> Storage Classes have a provisioner that determines what volume plugin is used

Reviewer: nit:
```suggestion
Each StorageClass has a provisioner that determines what volume plugin is used
```

### Reclaim Policy

> Persistent Volumes that are dynamically created by a StorageClass will have the

Reviewer: nit:
```suggestion
PersistentVolumes that are dynamically created by a StorageClass will have the
```

reclaim policy specified in the `reclaimPolicy` field of the class, which can be
either `Delete` or `Retain`. If no `reclaimPolicy` is specified when a
StorageClass object is created, it will default to `Delete`.

> Persistent Volumes that are created manually and managed via a StorageClass will have

Reviewer: ```suggestion
PersistentVolumes that are created manually and managed via a StorageClass will have
```

### Mount Options

> Persistent Volumes that are dynamically created by a StorageClass will have the

Reviewer: ```suggestion
PersistentVolumes that are dynamically created by a StorageClass will have the
```

## Parameters

> Storage Classes have parameters that describe volumes belonging to the storage

Reviewer: ```suggestion
StorageClasses have parameters that describe volumes belonging to the storage
```

  `"8452344e2becec931ece4e33c4674e4e,42982310de6c63381718ccfa6d8cf397"`. This
  is an optional parameter.
* `gidMin`, `gidMax` : The minimum and maximum value of GID range for the

>   StorageClass. A unique value (GID) in this range ( gidMin-gidMax ) will be

Reviewer: Fine; the original is also OK.

### Azure Disk

> #### Azure Unmanaged Disk StorageClass

Reviewer: I'd write:
```suggestion
#### Azure Unmanaged Disk storage class
```

  ignored. If a storage account is not provided, a new storage account will be
  created in the same resource group as the cluster.

> #### Azure Disk storage class (starting from v1.7.2)

Reviewer: tiny detail; if changing the heading I'd give it a stable anchor:
```suggestion
#### Azure Disk storage class (starting from v1.7.2) {#azure-disk-storage-class}
```

> #### New Azure Disk StorageClass (starting from v1.7.2)

Reviewer: For this heading, I recommend making a second change:
```suggestion
#### Azure Disk storage class (starting from v1.7.2) {#azure-disk-storage-class}
```

Reviewer: It might be an idea to revise the “Azure File” heading as well, to “Azure File storage class”.

* `pool`: The name of the StorageOS distributed capacity pool to provision the
  volume from.  Uses the `default` pool which is normally present if not specified.
* `description`: The description to assign to volumes that were created dynamically.
  All volume descriptions will be the same for the StorageClass, but different

>   Storage Classes can be used to allow descriptions for different use cases.

Reviewer: The original text works better, I think.
